### PR TITLE
Feature/world db docker compose

### LINKED ISSUE #2: Create a osm world database docker container
- [x] docker-compose setup
- [x] ensure data consistency in case of container shutdown
- [x] updateable database
- [x] updateable world file
- [x] documentation

### PR DESCRIPTION
**Pull request by hixi**
initial setup

reviewed by:
- [x] @das-g

Closes #2

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    - osm_world-db-postgis-data:/var/lib/postgresql
+  environment:
+    - POSTGRES_DB=gis
+    - POSTGRES_USER=gis
+    - POSTGRES_PASSWORD=gis
+
+osm_importer:
+  image: geometalab/osm2pgsql:latest
+  # for development, disable the line above and uncomment the line below
+  # build: .
```
Comment (das-g) [inline docker-compose.yml:25]: :+1:

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,43 @@
+FROM debian:jessie
+
+# osm2pgsql dependencies
```
Comment (das-g) [inline Dockerfile]: are all the dependencies listed here osm2pgsql dependencies or are some required for (compiling) osmupdate?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,26 @@
...
+
+Update duration, and thus the update interval, strongly depends on the hardware used.
+
+## Usage
+
+Using docker-compose:
+```
+docker-compose pull
+docker-compose run osm_importer
+```
```
Comment (das-g) [inline README.md:39]: maybe also document how to stop or suspend the repeated updating

**Comment by das-g** [comment]
Review done. Please see changes and comments.

File: Dockerfile (hunk 2 above)
Comment (hixi) [inline Dockerfile]: no, they are merged - thanks.

File: README.md (hunk 3 above)
Comment (hixi) [inline README.md:39]: good idea - it's wrong anyway. It should be something like `docker-compose up -d osm_importer`...

**Pull request merged**